highfidelity/hifi
# 8270 Changing avatars while sitting - restore animations

This PR restore the overwritten animation roles when avatar changes. 

https://highfidelity.fogbugz.com/f/cases/8270
...

## TEST PLAN
- Make sure your current avatar is the default avatar.
- Go to Dev-Welcome, find a place to sit and sit down.
- Open the tablet and go to market/avatars.
- Select a new avatar while sited. (bug repro with Photo-Real Male Avatar, but should work with anyone)
- Make sure everything looks normal. (no weird poses or lock animations)
